Update talk_basics in user manual (#11020)
nextcloud/documentation: user_manual/talk/talk_basics.rst

.. image:: images/chat-with-one-user.png

> If you want to create a custom group conversation, click the ``⋮`` button next to the filters button and then on ``create a new conversation``. 

Reviewer: ```suggestion
If you want to create a custom group conversation, click the ``⋮`` button next to the filters button and then on ``Create a new conversation``. 
```

.. image:: images/create-new-conversation.png
    :width: 300px

> You can then pick a name for the conversation and select if the conversation should be open to external users and if other users on the server can see and join the conversation.

Reviewer 2: ```suggestion
You can then pick a name for the conversation, put a description, and set up an avatar for it (with uploaded photo or emoji), and select if the conversation should be open to external users and if other users on the server can see and join the conversation.
```

.. image:: images/participant-menu.png

Moderators can configure the conversation. Select ``Conversation settings`` from the ``...`` menu of the conversation on the top to access the settings.

.. image:: images/open-settings.png
    :width: 300px

Here you can configure the description, guest access, if the conversation is visible to others on the server and more.

> .. image:: images/chat-room-settings.png

Reviewer 2: This screenshot is outdated

You can access your settings and choose a different webcam, microphone and other settings in the ``...`` menu in the top bar.

.. image:: images/media-settings.png
    :width: 300px

.. image:: images/devices-settings.png
    :width: 450px

> You can access other settings in ``Conversation settings``.

Reviewer 2: ```suggestion
You can change other settings in the ``Conversation settings`` dialog.
```

> .. image:: images/talk-settings.png

Reviewer 2: This screenshot also could be updated, as we have a couple of new sections in this list

You can access your settings and choose a different webcam, microphone and other settings in the ``...`` menu in the top bar.

.. image:: images/media-settings.png
    :width: 300px

> .. image:: images/device-settings-before-call.png

Reviewer 2: You should put that image before `images/join-call.png`, and set a small description, i think:

```
When you're in a conversation, you can start a call any time with the ``Start call`` button. Other participants will get notified and can join the call. Before call is started, you can select and switch on/off media devices (webcam, microphone) and select a virtual background

.. image:: images/device-settings-before-call.png
:width: 300px

 If somebody else has started a call already, the button will change in a green ``Join call`` button.
.. image:: images/join-call.png
```

    :width: 300px

or change the background 

.. image:: images/background-settings-in-call.png
    :width: 300px

> You can change other settings in the ``Conversation settings`` dialog.

Reviewer 2: This is a screenshot for `Talk settings` (bottom left corner)
